modules/logging-6x-v-5x.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+
+
+:_mod-docs-content-type: PROCEDURE
+[id="logging-6x-v-5x_{context}"]
+= OpenShift Logging 6.0: Key Differences from Previous Versions
+
+OpenShift Logging 6.0 is the culmination of efforts to modernize the underlying components of the logging solution. Over the past few releases, Loki and Vector were gradually introduced with a focus on improving performance and enhancing features while maintaining the stability required for business continuity.
+
+
+* **Horizontal Scalability:** Loki's architecture allows for seamless horizontal scaling, ensuring that the logging system can handle the growing demands of large-scale OpenShift deployments.
+
+* **Improved Query Performance:** Loki's indexing and query mechanisms are optimized for speed, enabling faster log retrieval and analysis.
+
+* **Streamlined Configuration:** The removal of Elasticsearch and Fluentd simplify setup and configuration.
+
+* **Improved Integration:** Loki and Vector offer tighter integration with OpenShift, providing a more seamless and native logging experience.
+
+* **Expanded Log Sources:** Vector's flexibility allows you to collect logs from an even wider range of sources, providing greater visibility into your OpenShift environment.
+
+* **Enhanced Log Processing:** Vector's processing capabilities have been expanded, enabling you to perform more complex filtering.
+
+* **OpenShift Web Console:** The integrated Logs page offers a user-friendly interface for searching, filtering, and viewing logs directly within the OpenShift console.
+
+* **Grafana:** OpenShift Logging integrates seamlessly with Grafana, enabling you to create rich, customizable dashboards to visualize and correlate log data with metrics and other observability data.
```

Comment:
Do we have existing docs that mention Grafana?  My inclination is to not mention it at all since we do not provide, intentionally a productized version.  My concern is the expectation this may introduce to customers

Reply:
I'll remove it.